api/src/main/java/jakarta/config/Configuration.java

```diff
@@ -25,13 +25,47 @@
 import java.lang.annotation.Target;
 
 /**
- * <p>
- *     The annotation to be used to mark the interface to be handled by the injection support.
- * </p>
+ * <p>Indicates that the annotated class is a <em>configuration interface class</em> to be handled by the Jakarta Config
+ * implementation injection support.</p>
+ *
+ * <p>The <em>configuration interface class</em> is resolved with portion of application's
+ * <em>persistent configuration</em> identified by <em>configuration path</em>.
+ *
+ * <p>The terms <em>configuration interface class</em>, <em>configuration key</em>, <em>configuration path</em>, <em>canonical
+ * representation</em>, <em>persistent configuration</em>, <em>resolve</em>, and others are used here as defined in
+ * the Jakarta Config specification.</p>
+
+ * <p><strong>\u26A0 Caution:</strong> you are reading an incomplete draft specification that is subject to change.</p>
  */
 @Target({ElementType.TYPE})
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
 public @interface Configuration {
 
+    /**
+     * The <em>configuration path</em> may identify where, in a given application's <em>persistent configuration</em>,
+     * the configuration relevant for the annotated configuration class may be found.
+     *
+     * <p>The configuration path is a <em>canonical representation</em> of a <em>configuration key</em>,
+     * as defined in the Jakarta Config specification.</p>
+     *
+     * <p>The configuration path represented by this element is effectively a hint to a Jakarta Config implementation
+     * about where, in an application's persistent configuration, a portion of that configuration relevant to the
+     * annotated configuration interface class may be found.</p>
+     *
```

Comment:
I suggest you add an example here to explain further instead of having the above 3 lines.